learn/analytics/bind_events_user.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Bind analytics events to a user
+description: This guide shows you how to manually bind analytics events to search queries using the X-MS-USER-ID HTTP header.
+---
+
+# Bind analytics events to a user
+
+By default, Meilisearch associates analytics events with the most recent search of the user who triggered them. This guide shows you how to manually bind analytics events to search queries using the `X-MS-USER-ID` HTTP header.
+
+## Requirements
+
+- A Meilisearch Cloud project with analytics and monitoring enabled
+- A working pipeline for submitting analytics events
+
+## Add `X-MS-USER-ID` to your search query
+
+Include the `X-MS-USER-ID` header in your search requests:
+
+<CodeSamples id="analytics_event_bind_search_1" />
+
+Replace `MEILISEARCH_USER_ID` with any value that uniquely identifies that user. This may be an authenticated user's ID when running searches from your own back end, or a user's IP address.
```

Comment:
```suggestion
Replace `X-MS-USER-ID` with any value that uniquely identifies that user. This may be an authenticated user's ID when running searches from your own backend, or a hash of the user's IP address.
```

Reply:
`MEILISEARCH_USER_ID` is referring to the placeholder value in the example. Also, although "backend" is accepted by some dictionaries, it seems most prefer "back end" and that's what we're using in the docs.

I'm modifying your suggestion directly instead of making a new one in this comment.